internal/ui/src/components/date-time/components/time-input.tsx

```diff
@@ -0,0 +1,249 @@
+// Reference: https://github.com/supabase/supabase/blob/master/apps/studio/components/ui/DatePicker/TimeSplitInput.tsx
+import { Clock } from "lucide-react";
```

Comment:
If you have an access to Nucleo icons let's pick a decent Clock from there. Possibly using `outlined` and `18px` then add it into `icons/pkg`